omniport/core/open_auth/serializers/application.py

```diff
@@ -215,3 +215,38 @@ def update(self, instance, validated_data):
         application = super().update(instance, validated_data)
 
         return application
+
+
+class ApplicationHiddenDetailSerializer(serializers.ModelSerializer):
+    """
+    Serializer for Application objects that exposes all information
+    """
+
+    team_members = PersonRelatedField(
```

Comment:
Not needed since you are not including this in the `fields` option. And also, this API just needs to serve the `client_secret`.